examples/gno.land/r/demo/postit/json.gno

```diff
@@ -0,0 +1,188 @@
+package postit
```

Comment:
What about adding a Render() func too, could just give some basic info and suggest to use the better frontend.